client/landing/jetpack-cloud/routes.js

```diff
@@ -19,23 +18,70 @@ import { jetpackCloudScanHistory } from './sections/scan-history/controller';
 import { jetpackCloudSettings } from './sections/settings/controller';
 
 const router = () => {
-	page( '*', normalize );
-
 	page( '/', setupSidebar, jetpackCloudDashboard, makeLayout, clientRender );
 
 	page( '/backups', siteSelection, setupSidebar, jetpackCloudBackups, makeLayout, clientRender );
-	page( '/backups/:site', siteSelection, setupSidebar, jetpackCloudBackups, makeLayout, clientRender );
```

Comment:
FYI: This got reformatted by a git hook when I was pushing the code. It only obfuscates the diff, so please disregard any changes from here on through the end of this file 🙂